packages/reporter/src/hooks/hooks.tsx

```diff
@@ -42,11 +48,22 @@ export interface HooksProps {
   model: HooksModel
 }
 
-const Hooks = observer(({ model }: HooksProps) => (
-  <ul className='hooks-container'>
-    {_.map(model.hooks, (hook) => <Hook key={hook.id} model={hook} />)}
-  </ul>
-))
+const Hooks = observer(({ model }: HooksProps) => {
+  const hooksCount: { [key: string]: number } = {
+    'before all': 0,
+    'before each': 0,
+    'after all': 0,
+    'after each': 0,
+  }
+
+  _.map(model.hooks, (hook) => hook.name !== 'test' && hooksCount[getHookName(hook.name)]++)
```

Comment:
Instead of `_.map`, I'd prefer to see `_.each` in cases like this, since it indicates it's mutating values and not collecting values into an array.